src/include/cell_inline.h

```diff
@@ -939,10 +939,10 @@ __wt_cell_unpack_safe(WT_SESSION_IMPL *session, const WT_PAGE_HEADER *dsk, WT_CE
                 WT_ASSERT(session,
```

Comment:
online 931 as hari mentioned, only checking the txn id is not enough.

Reply:
added comment